spinnaker-monitoring-daemon/spinnaker-monitoring/metric_filter.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Filter spectator metrics
+
+This filter is specified with a yaml file.
+
+There are three ways to filter, in the following order
+
+meters:
+  explicit:
+    # Each explicit metric name will be included
+    # If includeTags is present, then exclude all tags but those mentioned.
+    # If excludeTags is present, then exclude the mentioned tags.
+    # These are optional and independent of one another.
+    <explicit metric name>:
+       includeTags: <list>
+       excludeTags: <list>
+
+  # If metric was not explicitly enumerated in "explicit" dict, then
+  # check the metrics metadata against the following lists of regexes
+  - includeNameRegex: <value>
```

Comment:
The `-` marks this as a list entry under `explicit`. Is this the structure, or is `includeNameRegex` another distinct entry under `explicit` or `meters`?